### ISSUE TITLE
Restore on screen menu

### ISSUE THREAD

**Issue by eth101** [jimgraygit/Onion#43]
Is it possible to restore the on screen menu that comes with the stock launcher/os that's displayed with just pressing the menu button alone?

**Comment by ossidodicromo3** [comment]
Yes, the shortcut are useful, but managing the savestates through the stock menu is way more convenient than the text list

**Comment by maxengel** [comment]
agree with the above. the built-in save state manager is really simple. I'd prefer to avoid having to dive into RetroArch menus to manage save states and exit the emulator.

**Comment by denverslair** [comment]
Agree 😊

**Comment by xwu36** [comment]
+1, saving/loading via the stock menu is much easier.

**Comment by thias** [comment]
It's possible, but that original menu seems to actually be something hardcoded into each emulator (and a RetroArch build) by Miyoo. So for Onion where everything is RetroArch/libretro based, you need to launch that ra32.ss RetroArch binary. I've explained it here: https://github.com/jimgraygit/Onion/issues/56#issuecomment-1038283657

The menu is harcoded in the binary, so it's not themed.

**Comment by Sichroteph** [comment]
The auto save feature is the most pick up and play feature you can experience.
You launch the game were you were, menu + start to exit and save.

Now if you want to manage states, the retroarch states manager is direct accessible by pressing menu + select. Not in an obscure sub menu. 
From there, we found that what retroarch brings is immensively more better than the stock menu. States verionning, undo last state loaded etc. In addition, the snall drawback of loosing the menu allowed us to implement hotkeys. (menu + R2, menu + L2 etc.)
This logic is tied up to the Onion, to a point where in V3 it will breaks things if you try to change it.
